.halcyon/sandbox-extra-apps

```diff
@@ -0,0 +1,3 @@
+alex-3.1.3
+happy-1.19.4
+yesod-bin-1.4.3.1
```

Comment:
No, because the sets of dependencies for your app and for each extra app may be completely disjoints.

When Halcyon installs extra apps, it makes sure none of their dependencies contaminate your app’s sandbox — in contrast to what happens when you just use `cabal install`.